src/main/kotlin/no/nav/familie/ks/sak/integrasjon/distribuering/DistribuerVedtaksbrevTilVergeEllerFullmektigTask.kt

```diff
@@ -0,0 +1,50 @@
+package no.nav.familie.ks.sak.integrasjon.distribuering
+
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.familie.ks.sak.api.dto.DistribuerBrevDto
+import no.nav.familie.ks.sak.config.BehandlerRolle
+import no.nav.familie.ks.sak.kjerne.brev.BrevService
+import no.nav.familie.ks.sak.task.nesteGyldigeTriggertidForBehandlingIHverdager
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import org.springframework.stereotype.Service
+import java.util.Properties
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = DistribuerVedtaksbrevTilVergeEllerFullmektigTask.TASK_STEP_TYPE,
+    beskrivelse = "Send vedtaksbrev til manuelt registrert verge/fullmektig til Dokdist",
+    maxAntallFeil = 3,
+)
+class DistribuerVedtaksbrevTilVergeEllerFullmektigTask(
+    private val brevService: BrevService,
+) : AsyncTaskStep {
+    override fun doTask(task: Task) {
+        val distribuerBrevDto = objectMapper.readValue(task.payload, DistribuerBrevDto::class.java)
+        brevService.prøvDistribuerBrevOgLoggHendelse(
+            journalpostId = distribuerBrevDto.journalpostId,
+            behandlingId = distribuerBrevDto.behandlingId,
+            loggBehandlerRolle = BehandlerRolle.SYSTEM,
```

Comment:
Ønsker vi at det alltid er system her?

Reply:
Det er det som alltid har vært brukt ved utsending av vedtaksbrev, i hvertfall, og så er det SAKSBEHANDLER ved utsending av manuelle brev. Tror ikke det påvirker så mye annet enn hvilket ikon som vises for historikkinnslaget i frontend